README.md

```diff
@@ -2,7 +2,11 @@
   <a href="https://www.chartjs.org/" target="_blank">
     <img src="https://www.chartjs.org/media/logo-title.svg" alt="https://www.chartjs.org/"><br/>
   </a>
-    Simple yet flexible JavaScript charting for designers & developers
+  <a href="https://github.com/chartjs/Chart.js/actions?query=workflow%3ACI+branch%3Amaster">
+  <img alt="GitHub Workflow Status" src="https://img.shields.io/github/actions/workflow/status/chartjs/Chart.js/ci.yml?branch=master&style=flat-square">
```

Comment:
Not sure if we want to be dependant on a third party site in the readme, but if so the formatting seems incorrect